plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/panels/managers/BuildProgressSplitterPanelManager.kt

```diff
@@ -208,19 +209,14 @@ class BuildProgressSplitterPanelManager(private val project: Project) :
             }
         }
         val updatedStatuses = when (newState) {
-            TransformationStatus.CREATED, TransformationStatus.ACCEPTED -> {
+            TransformationStatus.CREATED, TransformationStatus.ACCEPTED, TransformationStatus.STARTED -> {
                 loadingPanelText = message("codemodernizer.toolwindow.scan_in_progress.accepted")
-                statuses.update(BuildStepStatus.WORKING, ProgressStepId.ACCEPTED)
```

Comment:
CREATED and ACCEPTED previously have BuildStepStatus.WORKING, but now will be BuildStepStatus.DONE. I think CREATED and ACCEPTED should still be WORKING?

Reply:
Default progress data should take care of showing the first step with the spinning / "working" icon at the beginning: https://github.com/aws/aws-toolkit-jetbrains/blob/6acda9854644267a1a5ef86c2ac5d6f7ab024c6c/plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/panels/managers/BuildProgressSplitterPanelManager.kt#L86

When I put `CREATED` and `ACCEPTED` in the `WORKING` state the icons do not update correctly:

![image](https://github.com/aws/aws-toolkit-jetbrains/assets/60020664/70e7ef5a-845b-4c4f-8a79-f3ed2eed09b1)

This makes sense also because once the job gets in the `CREATED` state I want to mark the upload as `DONE`